erpnext/accounts/utils.py

```
@@ -2724,3 +2724,129 @@ def build_qb_match_conditions(doctype, user=None) -> list:
+	if needs_stock:
+		pass
+
+	if cfg.get("check_credit_limit"):
+		_check_credit_limit_warn(doc)
+
+
+def _check_prev_docstatus(doc):
+	try:
+		if hasattr(doc, "check_prev_docstatus"):
+			doc.check_prev_docstatus()
+	except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)